pkg/generators/snippets/setters.go

```diff
@@ -3,19 +3,22 @@ package snippets
 import (
 	"fmt"
 
+	"github.com/kanopy-platform/code-generator/pkg/generators/tags"
 	"k8s.io/gengo/generator"
 	"k8s.io/gengo/types"
 )
 
 type Setter struct {
-	Root   *types.Type
-	Parent *types.Type
+	Root              *types.Type
+	Parent            *types.Type
+	IsPointerReceiver bool
```

Comment:
```suggestion
	pointerReceiver bool
```
it is not necessary to export this member.

Reply:
Thanks. I ended up removing this member altogether by using `defaultGeneratorArgs` as suggested from your other comment.